stl/inc/vector

```diff
@@ -3329,6 +3331,8 @@ public:
 
     template <_Container_compatible_range<bool> _Rng>
     constexpr void assign_range(_Rng&& _Range) {
+        static_assert(_STD assignable_from<bool&, _RANGES range_reference_t<_Rng>>,
+            "bool& must be assignable from the range's reference type. N4993 [sequence.reqmts]/60");

```

Comment:
I think I'd prefer we consistently say "Elements" here instead of `bool&`.

Reply:
Sure, I will follow your preference. I just considered that `vector<bool>` is *slightly* strange and wanted to be unambiguous about the target type of the assignment in the error message.

Reply:
My thinking is that the message should be as simple as we can make it, the expression is right there to provide all the specificity you'd ever want. I was tempted to suggest "Elements of the container must be assignable from elements of the range" or "Elements of the range must be assignable to elements of the container" but figured that was a bit too much.